ObjectiveGit/GTRepository.m

```diff
@@ -290,7 +290,10 @@ - (id)lookUpObjectByGitOid:(const git_oid *)oid objectType:(GTObjectType)type er
 		if (error != NULL) {
 			char oid_str[GIT_OID_HEXSZ+1];
 			git_oid_tostr(oid_str, sizeof(oid_str), oid);
-			*error = [NSError git_errorFor:gitError description:@"Failed to lookup object %s in repository.", oid_str];
+			*error = [NSError git_errorFor:gitError
+							   description:@"Failed to lookup object"
```

Comment:
You are right. Please remove the multi line error and make it one line. I looked up the other usages of this error method and it is a one liner everywhere. #StickingToTheStyleGuide